gallery/src/components/NxFilterInput/NxFilterInputPage.tsx

```diff
@@ -40,33 +44,64 @@ const NxFilterInputPage = () =>
             <td className="nx-cell">value</td>
             <td className="nx-cell">string</td>
             <td className="nx-cell">Yes</td>
-            <td className="nx-cell">Value to render in the input</td>
-          </tr>
-          <tr className="nx-table-row">
-            <td className="nx-cell">placeholder</td>
-            <td className="nx-cell">string</td>
-            <td className="nx-cell">No</td>
-            <td className="nx-cell">Placeholder to render in the input</td>
+            <td className="nx-cell">The value rendered in the text input</td>
           </tr>
           <tr className="nx-table-row">
-            <td className="nx-cell">className</td>
-            <td className="nx-cell">string</td>
+            <td className="nx-cell">onChange</td>
+            <td className="nx-cell">Function ((string) => void)</td>
             <td className="nx-cell">No</td>
-            <td className="nx-cell">Classes to apply to the element</td>
+            <td className="nx-cell">
+              A callback for when the user changes the value of the text box (e.g. by typing a letter)
+            </td>
           </tr>
           <tr className="nx-table-row">
-            <td className="nx-cell">inputId</td>
-            <td className="nx-cell">string</td>
+            <td className="nx-cell">onKeyPress</td>
+            <td className="nx-cell">Function ((string) => void)</td>
             <td className="nx-cell">No</td>
-            <td className="nx-cell">Id to assign to the input part of this component</td>
+            <td className="nx-cell">
+              <p className="nx-p">
+                A callback for when the user presses a key that doesn't necessarily change the input value
+                (e.g. by hitting enter)
+              </p>
+              <p className="nx-p">
+                The value given to the callback will be that of the key name, as described in the spec
+                for{' '}
+                <a target="_blank"
+                   rel="noopener"
+                   href="https://www.w3.org/TR/uievents-key/#named-key-attribute-values">
+                  named keys
+                </a>
+              </p>
+            </td>
           </tr>
           <tr className="nx-table-row">
-            <td className="nx-cell">onChange</td>
-            <td className="nx-cell">Function</td>
+            <td className="nx-cell">Input HTML Attributes | Textarea HTML Attributes</td>
```

Comment:
Unless there's a place where an `NxFilterInput` could be a `textarea` I'd recommend removing references to `Textarea`'s here and below, to me it's unnecessary and a little confusing.

Reply:
> Unless there's a place where an NxFilterInput could be a textarea I'd recommend removing references to Textarea's here and below, to me it's unnecessary and a little confusing.

2839e94e